packages/swarm-components/src/testUtils/screenRenderer.js

```diff
@@ -71,7 +69,7 @@ class screenRenderer {
 		return this;
 	}
 
-	createRoute(slug, element, bodyStyle = '') {
```

Comment:
I don't feel that we should kill `bodyStyle` here, it is still useful to adjust document style as a whole even if individual tests might want to use `wrapperStyle`.